src/main/java/seedu/module/logic/commands/FindModuleCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.module.logic.commands;
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.module.commons.core.Messages;
+import seedu.module.logic.commands.exceptions.CommandException;
+import seedu.module.model.Model;
+import seedu.module.model.task.Task;
+
+/**
+ * Finds and list all tasks in module book whose module code matches the module code provided.
+ */
+public class FindModuleCommand extends Command {
+    public static final String COMMAND_WORD = "mod";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Find tasks associated with "
+            + "the Module code provided and displays them as a list with index numbers.\n"
+            + "Parameters: MODULE_CODE \n"
+            + "Example: " + COMMAND_WORD + " CS2103T";
+
+    private final String moduleCode;
+    private final Predicate<Task> predicate;
+
+    /**
+     * Constructor of FindModuleCommand
+     * @param moduleCode
```

Comment:
An empty line between description and parameter section. According to CS2103T coding standard.
Similar issues in the rest of code.

Reply:
All resolved!